framework/validators/EmailValidator.php

```diff
@@ -24,13 +24,13 @@ class EmailValidator extends Validator
      * @var string the regular expression used to validate the attribute value.
      * @see http://www.regular-expressions.info/email.html
      */
-    public $pattern = '/^[a-zA-Z0-9!#$%&\'*+\\/=?^_`{|}~-]+(?:\.[a-zA-Z0-9!#$%&\'*+\\/=?^_`{|}~-]+)*@(?:[a-zA-Z0-9](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?\.)+[a-zA-Z0-9](?:[a-zA-Z0-9-]*[a-zA-Z0-9])?$/';
+    public $pattern = '/(*UTF8)^[\pL0-9!#$%&\'*+\\/=?^_`{|}~-]+(?:\.[\pL0-9!#$%&\'*+\\/=?^_`{|}~-]+)*@(?:[\pL0-9](?:[\pL0-9-]*[\pL0-9])?\.)+[\pL0-9](?:[\pL0-9-]*[\pL0-9])?$/';
```

Comment:
Isn't it better to use `u` modifier at the end of the pattern instead of inline `(*UTF8)`??